### ISSUE TITLE
Add option to `files_external:option` and `files_external:config` to remove a field

### ISSUE THREAD

**Issue by PanieriLorenzo** [nextcloud/server#35794]
### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are interested into the same feature.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments. 

## The Problem
Currently the `files_external:option` and `files_external:config` commands don't have a way to unset or remove a field. So if you accidentally add a field you have to do database trickery to remove it again.

## Proposal
Add an option `--remove FIELDNAME` or `--unset FIELDNAME` to remove a field.

**Comment by manumitted** [comment]
Yes, I was having a problem with exception because "port" is not an "int", but a "string" -- because it's an empty value.  I guess this used to work, but now it just makes Nextcloud show absolutely no files.  ugh. (disastrous / panic ensues)

```
TypeError fsockopen(): Argument #2 ($port) must be of type int, string given
```

Well, I put the "port" in the options and not the config.  So, now I have warnings about it.   Ugh again, but not exactly disastrous after I wised up and put it into the config.

```
[PHP] Warning: Undefined array key "port" at /var/www/nextcloud/apps/files_external/lib/Command/ListCommand.php#183
	from ? by -- at Feb 13, 2026, 3:13:05 PM
```
